src/modules/arrayFilter/arrayFilter.test.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { arrayFilter } from './../arrayFilter/arrayFilter';
+
+describe('Describe array map', () => {
```

Comment:
You only have one test, so you don't need to wrap your test in a "describe" block.

Reply:
Done.